engine/components-api/src/main/java/com/cloud/storage/StorageManager.java

```diff
@@ -124,7 +124,7 @@
             "Storage",
             "60",
             "Timeout (in secs) for the storage pool client connection timeout (for managed pools). Currently only supported for PowerFlex.",
-            true,
+            false,
```

Comment:
Just out of curiosity. Why is `STORAGE_POOL_CLIENT_TIMEOUT` changed from dynamic to static?

Reply:
> Just out of curiosity. Why is `STORAGE_POOL_CLIENT_TIMEOUT` changed from dynamic to static?

@GabrielBrascher This config is currently support for ScaleIO only. It's value is picked by the ScaleIO client when it is created, which is either on the primary storage addition or management server (re)start (for existing pools). In order to apply this config value for existing pools, it is required to restart the management server. Changing this config to 'static' prompts the operator, to restart the management server to apply the new value.

Reply:
Perfect, that makes sense. Thanks for the explanation, @sureshanaparti!